mPower2/mPower2.xcodeproj/project.pbxproj

```diff
@@ -824,7 +865,7 @@
 					F8D3E7752062F1B900CD351C = {
 						CreatedOnToolsVersion = 9.2;
 						LastSwiftMigration = 1020;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```

Comment:
Your development cert was expired for manual, so I switched it to automatic.  I prefer automatic for dev certs anyways, it's just easier to onboard devs.  Ok to keep?